sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/higherOrderFunctions.scala

```diff
@@ -291,11 +291,11 @@ case class ArrayTransform(
  */
 // scalastyle:off line.size.limit
 @ExpressionDescription(
-  usage = """_FUNC_(expr, func) - Sorts the input array in ascending order. The elements of the
-    input array must be orderable. Null elements will be placed at the end of the returned
-    array. Since 3.0.0 this function also sorts and returns the array based on the given
-    comparator function. The comparator will take two arguments
-    representing two elements of the array.
+  usage = """_FUNC_(expr, func) - Sorts the input array. If func is omitted, sort
+    in ascending order. The elements of the input array must be orderable. Null elements
+    will be placed at the end of the returned. Since 3.0.0 this function also sorts and
```

Comment:
Did you skip `array` in _Null elements will be placed at the end of the returned **array**._ intentionally?

Reply:
Good catch. I missed it.

Reply:
Also I realized "and and ...".